src/quo2/components/account_avatar.cljs

```diff
@@ -0,0 +1,48 @@
+(ns quo2.components.account-avatar
+  (:require [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [status-im.ui.components.icons.icons :as icons]))
+
+(def light-mode-color
+  {:dark colors/customization-purple-60
+   :light colors/customization-purple-50})
+
+(def sizes
+  {:xxl 80
+   :xl 48
+   :large 32
+   :medium 24
+   :small 20})
+
+(def border-radii
+  {:xxl 16
+   :xl 12
+   :large 10
+   :medium 8
+   :small 6})
+
+(def inner-icon-sizes
+  {:xxl 36
+   :xl 24
+   :large 15
+   :medium 11
+   :small 11})
+
+(defn account-avatar
+  [{:keys [size dark? icon]}]
```

Comment:
(same here) better to confirm with design team, but it looks like this component uses emoji and constructs icon in component

Reply:
also, `dark?` (use app theme)

Reply:
Done

Reply:
Thanks Parvesh .. I'll ask them before merging